cmd/kubeapps-apis/server/plugins_test.go

```diff
@@ -240,3 +243,120 @@ func createTestFS(t *testing.T, filenames []string) fstest.MapFS {
 	}
 	return fs
 }
+
+func TestExtractToken(t *testing.T) {
+	testCases := []struct {
+		name          string
+		contextKey    string
+		contextValue  string
+		expectedToken string
+		expectedErr   string
```

Comment:
You may also be able to use the `errors` package to instead define an error type here (ie. nil, for no error, or an actual error if it is, then use [errors.Is](https://golang.org/pkg/errors/#Is). But fine as is.

Reply:
I've changed it to `fmt.Errorf(...)` and `status.Errorf` (passing the expected `codes.Unauthenticated` instead of relying on the string msg). Thanks!